packages/toast/src/ControlledToast/ControlledToast.spec.tsx

```diff
@@ -69,6 +69,22 @@ describe('packages/toast/controlled', () => {
       expect(toast).toBeInTheDocument();
     });
 
+    test('handles rerender when `open` is true without hanging', async () => {
```

Comment:
I assume this test fails when we don't use `useStableControlledToastProps`?

Reply:
Yup! Or technically it just hangs in an infinite loop, but it would fail in Evergreen from a timeout eventually.